src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random numberOfColor = new Random();
```

Comment:
`Random` instance is not a number yet :)
```suggestion
    private final Random random = new Random();
```

Reply:
done